services/subscriber.py

```diff
@@ -24,13 +24,11 @@ def get_subscribers(self, targetings, hours_whitelist, volume):
                 "value": timezones
             })
         try:
-            s = Search(using=es, index="users")
-            s.extra(size=volume)
+            s = Search(using=es, index="subscribers")
```

Comment:
`index="users"` was correct, it is an alias

https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html